lib/engine/game/g_1835/round/draft.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative '../../../round/draft'
+
+module Engine
+  module Game
+    module G1835
+      module Round
+        class Draft < Engine::Round::Draft
+          def select_entities
+            @game.players
```

Comment:
This looks the same as the base class (assuming reverse is set to false)

Could you remove?

Reply:
there is a variant "Clemens", where reverse is true, not fully implented.